### PR TITLE
Add newsletter in home

### PR DESCRIPTION
**Pull request by shrinathprabhu**
# Resolves

- [AR-3436](https://team-1624093970686.atlassian.net/browse/AR-3436)

## Changes

- Clear newsletter field after success
- Add newsletter in home

## Screenshot
Screenshot from 2022-07-08 19-04-01

# Checklist

- [x] The branch name follows the format: `developer-name/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

### CODE REVIEW

**Review by karthik-durai — APPROVED** [review]

File: components/home/Newsletter.vue [hunk 1]
```diff
@@ -18,7 +18,7 @@
               size="6.4rem"
               gradient="linear-gradient(121.85deg, #C4C4C4 29.11%, #FFFFFF 69.16%)"
             >
-              1000’s
+              1000's
```
Comment (ajithranka) [inline components/home/Newsletter.vue:21]: Why was this changed (single curly quote to single straight quote)? Is this intentional or coming from a formatter?

I think we should prefer the curly quote for this usage. See this [guide](https://practicaltypography.com/straight-and-curly-quotes.html) for details.

**Review by ajithranka — APPROVED** [review]

File: components/home/Newsletter.vue (hunk 1 above)
Comment (shrinathprabhu) [reply components/home/Newsletter.vue:21]: Replaced it to be consistent with content on other places, In all our microsites and websites we are using single straight quote
